examples/src/main/kotlin/io/grpc/examples/helloworld/HelloWorldClient.kt

```diff
@@ -16,31 +16,22 @@
 
 package io.grpc.examples.helloworld
 
-import io.grpc.StatusException
 import io.grpc.ManagedChannel
 import io.grpc.ManagedChannelBuilder
 import io.grpc.examples.helloworld.GreeterGrpcKt.GreeterCoroutineStub
+import kotlinx.coroutines.*
```

Comment:
Can we keep the imports individual?

We haven't open-sourced the Kotlin style guide, but I'd prefer to keep this in Google style to avoid sync issues, and that requires the absence of wildcard imports (carried over from Java: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
Yeah. IntelliJ decided to collapse them.  Fixing...